### PR TITLE
Fix UI labels

### PR DESCRIPTION
**Pull request by vallmoy**

### CODE REVIEW

File: properties/world.py [hunk 1]
```diff
@@ -50,9 +50,9 @@ class AppleseedSkySettings(bpy.types.PropertyGroup):
     env_type = bpy.props.EnumProperty(name="Environment Type",
                                       items=[("constant", "Constant", "Use constant color for sky"),
                                              ("gradient", "Gradient", "Use sky color gradient"),
-                                             ("latlong_map", "Latitude-Longitude Map", "Use latlong map texture"),
-                                             ("mirrorball_map", "Mirror Ball Map", "Use mirror ball texture"),
-                                             ("sunsky", "Physical Sun/Sky", ""),
+                                             ("latlong_map", "HDRI Environment", "Use hdri map texture"),
```
Comment (dictoon) [inline properties/world.py]: *HDRI* (capital)

File: ui/objects.py [hunk 2]
```diff
@@ -99,9 +95,19 @@ def draw(self, context):
...
 
+        layout.prop(asr_obj, "medium_priority", text="Nested Glass Priority")
+
+        layout.prop(asr_obj, "ray_bias_method", text="Ray Bias")
+        row = layout.row()
+        row.enabled = asr_obj.ray_bias_method != 'none'
+        row.prop(asr_obj, "ray_bias_distance", text="Ray Bias Distance")
+
+        layout.prop_search(asr_obj, "object_sss_set", sss_lists, "sss_sets", text="SSS Set")
+        #layout.label(text="SSS sets are created in the world tab")
```
Comment (dictoon) [inline ui/objects.py]: Remove comment out code.

**Comment by vallmoy** [comment]
Updated!

**Comment by dictoon** [comment]
Thanks! Things are clearly moving in the right direction.

On a purely technical level, I would have preferred the commits to be squashed into a single one, rather than adding "Update XXX.py" commits which make following the code history and writing release notes much harder than necessary.

I'll rebase and squash them together now.

**Comment by dictoon** [comment]
Oh you did squash! Nevermind. Sorry for the noise!

**Pull request merged**